osrdyne/src/main.rs

```diff
@@ -98,8 +101,20 @@ async fn main() -> Result<(), anyhow::Error> {
     ));
 
     let (worker_list_send, woker_list_recv) = tokio::sync::watch::channel(Arc::new(vec![]));
```

Comment:
Is it a typo (not from your PR but can be easily fixed)?

```suggestion
    let (worker_list_send, worker_list_recv) = tokio::sync::watch::channel(Arc::new(vec![]));
```